### DISCUSSION TITLE
Weird behaviour with CombineMols

### DISCUSSION THREAD

**Discussion by EBjerrum** [rdkit/rdkit#7977] [General] [answered]
I have some unexpected behaviour when combining mols. Afterwards RDKit switches to V3000 for the molblock and It seems that there's a spurious collection (StereoGroup), that is not even readable. It gets ignored by RDKit but some other softwares choke on it. I would expect simply to get a V2000 molblock and no "ghost" collection.

```
man_frag = Chem.MolFromSmiles("N", sanitize=True)
man_frag2 = Chem.MolFromSmiles("C", sanitize=True)

out_combined = Chem.CombineMols(man_frag, man_frag2)
molblock = Chem.MolToMolBlock(out_combined)
print(molblock)
```

     RDKit          2D

  0  0  0  0  0  0  0  0  0  0999 V3000
M  V30 BEGIN CTAB
M  V30 COUNTS 2 0 0 0 0
M  V30 BEGIN ATOM
M  V30 1 N 1.000000 0.000000 0.000000 0
M  V30 2 C 0.000000 0.000000 0.000000 0
M  V30 END ATOM
**M  V30 BEGIN COLLECTION
M  V30 MDLV30/STEABS ATOMS=(0)
M  V30 END COLLECTION**
M  V30 END CTAB
M  END

Is there someone who can shed a bit of light on what might be happening?

**Comment by ptosco** [comment, marked as answer]
@EBjerrum Hi Esben, I have come across this myself ([#7782](https://github.com/rdkit/rdkit/issues/7782)) and fixed it ([#7783](https://github.com/rdkit/rdkit/pull/7783)). The fix is part of the latest `2024_09_2` release.

**Reply by EBjerrum** [reply]
Btw. Is there a way to remove the empty stereogroup? it doesn't seem to be there when I asked for GetStereoGroups?

**Comment by EBjerrum** [comment]
@ptosco Thanks!
